modules/tobira/src/main/java/org/opencastproject/tobira/impl/Item.java

```diff
@@ -373,12 +377,28 @@ private static Jsons.Val findTimelinePreview(MediaPackage mp) {
         Jsons.p("updated", series.getModifiedDate().getTime())
       );
     } else {
+      // Created date
+      var createdDateString = series.getDublinCore().getFirst(PROPERTY_CREATED);
+      var created = Jsons.NULL;
+      try {
+        var ta = DateTimeFormatter.ISO_INSTANT.parse(createdDateString);
```

Comment:
```
2024-02-15 12:49:27,600 | WARN  | (Item:387) - Series ID-openmedia-opencast has bad created-date:                                             
java.time.format.DateTimeParseException: Text '2024-02-15T18:36Z' could not be parsed at index 1
```

I hate date/time classes :)

Reply:
I think `DateTimeFormatter.ISO_INSTANT.parse()` can be replaced with `Instant.parse()` and then it should work.

Reply:
Uhg yeah, I hate them as well. I pushed a commit that should probably fix it. Using the same helper functions as other parts of Opencast.